btr-web/btr-common-components/components/bcros/inputs/EmailField.vue

```diff
@@ -21,6 +29,7 @@ defineProps({
   placeholder: { type: [String], default: '' },
   id: { type: String, required: true },
   modelValue: { type: String, default: '' },
-  variant: { type: String, default: 'bcGov' }
+  variant: { type: String, default: 'bcGov' },
+  translate: { type: Boolean, default: false }
```

Comment:
additionally $t() will try to find translation, if it does not, it will just display provided string. Which means, if we provide some error string e.g. "This is some error, please retry." It will display this exact text, as it will not be able to find translation.

Reply:
$t will display a warning in development environments however as a hint to add it to the file so we want to avoid that, we were able to remove this and use a custom error map in it's place